test/bdd/pkg/vc/vc_steps.go

```diff
@@ -329,14 +329,14 @@ func (e *Steps) retrieveCredential(profileName string) error {
 		return expectedStatusCodeError(http.StatusOK, resp.StatusCode, respBytes)
 	}
 
-	unescapedResponse, err := strconv.Unquote(string(respBytes))
-	if err != nil {
-		return err
-	}
-
 	expectedCredential := string(e.bddContext.CreatedCredential)
-	if unescapedResponse != expectedCredential {
-		return expectedStringError(expectedCredential, unescapedResponse)
+	receivedCredential := string(respBytes)
+	// For some reason there's an extra line at the end of the credential returned from the create credential handler.
```

Comment:
any idea why this is? Normally it doesn't matter since we're just using the JSON unmarshal functions, but it's still odd.

Reply:
Instead of appending a breakline, compare against trimmed expectedCredential, but definitely this needs to be investigated.

Reply:
Done!